bin/unit_test

```diff
@@ -48,7 +48,7 @@ if [[ $tests == *swift* ]]
 then
   cd swift/looker || exit
   echo "TODO: Unit testing Swift SDK ..."
-  # swift test
+  swift test --filter "api|authToken|constants|models|transport"
```

Comment:
too bad there's not a more general filter that would catch these all

Reply:
we can _probably_ move those into a `unit` subfolder in a subsequent PR. I think the pattern match will still work.